common/test/acceptance/tests/studio/test_studio_outline.py

```diff
@@ -1578,3 +1583,132 @@ def _get_items(self):
         unit = subsection.expand_subsection().unit(UNIT_NAME)
 
         return (section, subsection, unit)
+
+
+@attr('shard_3')
+class DeprecationWarningMessageTest(CourseOutlineTest):
+    """
+    Feature: Verify deprecation warning message.
+    """
+    HEADING_TEXT = 'This course uses features that are no longer supported.'
+    COMPONENT_LIST_HEADING = 'You must delete or replace the following components.'
+    ADVANCE_MODULES_REMOVE_TEXT = ('To avoid errors, edX strongly recommends that you remove unsupported features '
+                                   'from the course advanced settings. To do this, go to the Advanced Settings '
+                                   'page, locate the "Advanced Module List" setting, and then delete the following '
+                                   'modules from the list.')
+
+    def _add_deprecated_advance_modules(self):
```

Comment:
Can we pass list of modules `['peergrading', 'combinedopenended']` to make it more generic so we can use same method in future.